datasets/kwaliteitsmonitorwonen/dataset.json

```diff
@@ -0,0 +1,160 @@
+{
+  "id": "kwaliteitsmonitorwonen",
+  "type": "dataset",
+  "title": "Kwaliteitsmonitor Wonen",
+  "description": "Per verblijfsobject de kenmerken van de woningvoorraad van de gemeente Amsterdam.",
+  "version": "1.0.0",
+  "status": "beschikbaar",
+  "theme": [
+    "Wonen"
+  ],
+  "publisher": "OIS",
+  "owner": "Gemeente Amsterdam",
+  "keywords": [
+    "Wonen"
+  ],
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "kwaliteitsmonitorwonen",
+      "type": "table",
+      "version": "1.0.0",
+      "title": "Kwaliteitsmonitor Wonen",
+      "schema": {
+        "required": [
+          "id",
+          "schema"
+        ],
+        "display": "naam",
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "id": {
+            "type": "string",
+            "description": "Unieke aanduiding objecttype"
+          },
+          "vboIdentificatie": {
+            "type": "string",
+            "description": "Unieke identificatie van het verblijfsobject"
+          },
+          "vboGeometrie": {
+            "description": "Geometrie van het type POINT van het verblijfsobject",
+            "$ref": "https://geojson.org/schema/Point.json"
+          },
+          "vboOppervlakte": {
+            "type": "integer",
+            "description": "De oppervlakte van het verblijfsobject"
+          },
+          "vboGebruiksdoel": {
+            "type": "string",
+            "description": "Het gebruiksdoel van het verblijfsobject"
+          },
+          "pndIdentificatie": {
+            "type": "integer",
+            "description": "Unieke identificatie van het pand"
+          },
+          "pndGeometrie": {
+            "description": "Geometrie van het type MULTIPOLYGON van het pand",
+            "$ref": "https://geojson.org/schema/Multipolygon.json"
+          },
+          "pndGeometrieMiddelpunt": {
+            "description": "Geometrie van het middelpunt van het pand van het type POINT",
+            "$ref": "https://geojson.org/schema/Point.json"
+          },
+          "pndOorspronkelijkbouwjaar": {
+            "type": "integer",
+            "description": "Oorspronkelijke bouwjaar van het pand"
+          },
+          "pndLigging": {
+            "type": "string",
+            "description": "Ligging van het pand"
+          },
+          "mntMonumentstatus": {
+            "type": "string",
+            "description": "Monumentstatus van het pand"
+          },
+          "bsdLigtInBeschermdGebied": {
+            "type": "string",
+            "description": "Ligt het verblijfsobject in een beschermd gebied?"
```

Comment:
Nog een keer de vraag, wat zit hier in? De beschrijving suggereert dat dit een boolean zou moeten zijn,  of dat het veld hieronder voor sommige rijen een null zou moeten krijgen.

Reply:
Dit kan een string blijven.